content/rails/displaying-booleans-in-active-admin.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Displaying booleans in Active Admin
+date: 2015-08-13
+author: Josephine Hall
+---
+
+In Active Admin if you want to display a boolean propetry that doesn't directly map to a database column you can use `status_tag` to display the value in a fiendly way:
```

Comment:
Mind adding a link to the docs? I didn’t really understand what the `:ok` bit was about until reading them.